include/freerdp/freerdp.h

```diff
@@ -508,6 +508,7 @@ extern "C"
 	FREERDP_API const char* freerdp_get_logon_error_info_data(UINT32 data);
 
 	FREERDP_API ULONG freerdp_get_transport_sent(rdpContext* context, BOOL resetCount);
+	FREERDP_API void freerdp_set_transport_callbacks(rdpContext* context, void* io_callbacks);
```

Comment:
Sorry, missed this one previously, the argument should probably be `const rdpIoUpdate* io_callbacks` as this is the only type allowed, correct?

Reply:
yes.

Reply:
it's ok, anyway it should NOT be merged yet,  i need to review all changes and confirm what it is sufficient to affload whole io thing for both clients and servers first.